sw/device/lib/dif/register.c

```diff
@@ -0,0 +1,17 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/dif/register.h"
+
+// |extern| declarations to give the inline functions in the
+// corresponding header a link location.
+extern reg32_t reg32_from_addr(uintptr_t);
+extern uint32_t reg32_read(reg32_t, ptrdiff_t);
+extern void reg32_write(reg32_t, ptrdiff_t, uint32_t);
+extern void reg32_clear_mask(reg32_t, ptrdiff_t, uint32_t);
+extern void reg32_set_mask(reg32_t, ptrdiff_t, uint32_t);
```

Comment:
You need to update these signatures so they match the function definitions.

Reply:
Oops, will do